locales/en/public.json

```diff
@@ -141,23 +141,23 @@
     },
     "TITLE": "{{chartKind}} (last {{duration}}s, every {{period}}s)"
   },
-  "ClickableAutomatedAnalysisLabel": {
+  "ClickableAutomatedAnalysisresult": {
```

Comment:
Please put this key back to how it was previously - I don't think this string appears in the UI but is used for looking up the localized strings.

Reply:
I think this is a result of `yarn localize`. I don't remember manually updating this.

Reply:
Strange. I don't know why that would change the keys in the dictionary, but I suppose it could. Changing this on its own like this is likely to break things however. It would be good to figure out why it's changing this, but in any case I don't think the change should be included.